src/mlir/generator.cc

```diff
@@ -246,11 +316,17 @@ namespace mlir::verona
     if (nodes.size() == 1)
     {
       auto name = AST::getID(nodes[0]);
-      if (auto type = typeTable.lookup(name))
+      if (AST::isClassType(ast))
+      {
+        // Forward declare a class
```

Comment:
Bit worried about the forward declare comment?  We do not expect classes to need forward declaration.

Reply:
Ah, yes, I didn't mean "forward declaration" as in the language, but I agree the name overlap is unfortunate. I'll reword it.